src/style.css

```
@@ -1 +1,66 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Zen+Antique+Soft&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  background: #fff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-align: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
+
+.nav_item {
+  display: inline-block;
+  position: relative;
+  height: 34px;
+  margin: 0 10px;
+}
+
+.item_last {
+  padding-right: 40px;
+}
+
+.logo {
+  padding: 10px 0 10px 50px;
+  left: 0;
+}
+
+.nav_link {
+  display: block;
+  padding: 22.5px 0;
+  text-decoration: none;
+  font-family: "Roboto", "Arial", sans-serif;
+  font-style: normal;
+  color: #000;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  width: 100%;
+  position: absolute;
+  left: 0;
+  top: 56px;
```

```suggestion
  bottom: 0;
```
if the height of links becomes higher? you just need it to be always at the bottom of a link